yt_dlp/extractor/ximalaya.py

```diff
@@ -123,14 +123,20 @@ def _real_extract(self, url):
 class XimalayaAlbumIE(XimalayaBaseIE):
     IE_NAME = 'ximalaya:album'
     IE_DESC = '喜马拉雅FM 专辑'
-    _VALID_URL = r'https?://(?:www\.|m\.)?ximalaya\.com/\d+/album/(?P<id>[0-9]+)'
+    _VALID_URL = r'https?://(?:www\.|m\.)?ximalaya\.com/(?:\d+/)?album/(?P<id>[0-9]+)'
     _TESTS = [{
         'url': 'http://www.ximalaya.com/61425525/album/5534601/',
         'info_dict': {
             'title': '唐诗三百首（含赏析）',
             'id': '5534601',
         },
         'playlist_mincount': 323,
+    }, {
+        'url': 'https://www.ximalaya.com/album/6912905',
+        'info_dict': {
+            'title': '埃克哈特《修炼当下的力量》',
+            'id': '6912905',
+        }
```

Comment:
for playlists it's good to test the number of entries too
```suggestion
        },
        'playlist_mincount': 41,
```

Reply:
Not just good, one of the `playlist_*` keys is mandatory